Assets/Scripts/PlanetAnimalCountTEMP.cs

```diff
@@ -1,50 +1,57 @@
-using System.Collections;
+﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 
 public class PlanetAnimalCountTEMP : MonoBehaviour
 {
-    private int calmCaught = 0;
-    private int calmSpawned;
-    private TMPro.TextMeshProUGUI calmUI;
+    private List<GameObject> panels = new List<GameObject>(); //the grid of panels that the animals will spawn as a child on
+    public List<GameObject> collectedAnimals = new List<GameObject>();
+    private int lastSpace;
 
-    private int hostileCaught = 0;
-    private int hostileSpawned;
-    private TMPro.TextMeshProUGUI hostileUI;
-
-    private int scaredCaught = 0;
-    private int scaredSpawned;
-    private TMPro.TextMeshProUGUI scaredUI;
+    //prefabs
+    private GameObject neutral;
+    private GameObject hostile;
+    private GameObject scared;
 
     //public SpawningManager spawningManager;
-    public static PlanetAnimalCountTEMP	Instance;
+    public static PlanetAnimalCountTEMP Instance;
+    public static AnimalController animalController;
 
     // Start is called before the first frame update
     void Start()
     {
         Instance = this;
-        calmUI = GameObject.Find("Calm Caught").GetComponent<TMPro.TextMeshProUGUI>();
-        hostileUI = GameObject.Find("Hostile Caught").GetComponent<TMPro.TextMeshProUGUI>();
-        scaredUI = GameObject.Find("Scared Caught").GetComponent<TMPro.TextMeshProUGUI>();
-    }
 
-    public void AnimalCount(string name) //Sent from AnimalController.cs
-    {
-        if (name.Contains("Neutral"))
+        for (int i = 0; i < 25; i++)
```

Comment:
Can the 25 be a const instead of a magic number for better readability and robustness.

Reply:
Boom. Done.